### ISSUE TITLE
[Suggestion] Bundling groups together. (Tridaak)

### ISSUE THREAD

**Issue by Proxying** [Proxying/TabManager#1]
Add options onto a permission plugin along with tabmanager.group.name, that states what order groups should be displayed as.

```
Ie:
admin group:
tabmanager.group.admin = true
tabmanager.order = 1

donator group:
tabmanager.group.donator = true
tabmanager.order = 5

default group:
tabmanager.group.default = true
tabmanager.order = 5
```
Then when the player opens the tab list, it displays users in the admin group first (since it has highest ranking for order) and all other players organized randomly from there (since they have same ranking).

In the end it should work similarly to how mineplex handles different ranks in the tablist: https://i.gyazo.com/efbafa85828bf9702384f241531fa86c.png

**Comment by SubtypeZero** [comment]
It might be better to have configurable tab "weights". I'm not quite sure how the meta system works, but it would be nice to be able to set that in group meta with LuckPerms.

**Comment by Mikeboi** [comment]
> To “group” players in the Tab List by their permission group (via Luckperms, ZPermissions, whatever), enable “Add Players to Teams” in your main tabmanager.conf file. This will automagically enable grouping with other players of the same group.

i tried doing this but it doesnt seem to work, is another step required on Luckperm side?

**Comment by Proxying** [comment]
No, but if you have other plugins that alter or add players to teams it may cause conflicts. The ordering is done by each client individually, so client-side mods that change their tab list (maps etc) may also cause adverse effects.
